src/hotspot/share/opto/runtime.hpp

```diff
@@ -216,7 +215,6 @@ class OptoRuntime : public AllStatic {
   static address slow_arraycopy_Java()                   { return _slow_arraycopy_Java; }
   static address register_finalizer_Java()               { return _register_finalizer_Java; }
 #if INCLUDE_JVMTI
-  static address notify_jvmti_object_alloc()             { return _notify_jvmti_object_alloc; }
```

Comment:
This line has to be also removed:
`312   static const TypeFunc* notify_jvmti_object_alloc_Type();`